### DISCUSSION TITLE
Trouble calculating LRMES

### DISCUSSION THREAD

**Discussion by LINNEAZACKRISSON** [TommasoBelluzzo/SystemicRisk#22] [Q&A]
Hello!
I would like to start by saying that I'm a big fan of your page!
I have some problems calculating LRMES. If I set beta to be 1,5 and d=0,4, then I get LRMES to be 0,2831. This seems very low? What can i be doing incorrect?
Best

**Comment by TommasoBelluzzo** [comment]
Hi @LINNEAZACKRISSON, thank you for the feedback!

Are you using my package or your own implementation? In the latter case I cannot help you unless you share some code.
Maybe an error in the LRMES computation? According to my formulation, for given d=0.4 and b=1.5, LRMES should be close to 0.5.

```
d = 0.4;
beta_interval = [0.0 3.0];

lrmes_func = @(x)1.0 - exp(log(1.0 - d) * x);

fplot(lrmes_func, beta_interval);
```

**Reply by LINNEAZACKRISSON** [reply]
Thank you very much! I got it right now! 
I have another perhaps stupid question… but is this some sort of simulation of the LRMES? Or what would you call it?

**Reply by TommasoBelluzzo** [reply]
It's a closed form approximation, we might call it like that.

**Reply by tcc27** [reply]
Hi, sorry to bother you. I would like to ask if we have a reference for this approximation formula of  LRMES ? Because I didn't find this formula(1 - exp(log(1 - d) .* beta)) in Brownlees & Engle (2010)
